index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Setup Mobile Project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <article>
+        <header>   
+            <time class="time"><b>9:41</b></time>
+        <span>
+            <img class="menu-icon" src="/images/Icon - Menu.svg" alt="hamburger list">
+            <img class="pwr" src="/images/Battery.svg" alt="battery">
+            <img class="wifi" src="/images/Wifi.svg" alt="wifi">
+            <img class="cell-connection" src="/images/Cellular Connection.svg" alt="cell connection">
+        </span>
```

Comment:
- Good job trying to stick to the design template as closely as possible, but I suggest you do not need to include this in your design. They were used in the Figma template to signify a mobile design. Please remove this 😄

Reply:
Thank you for the feedback, implemented the recommendation as suggested.